deployments/deploy.go

```diff
@@ -73,3 +68,29 @@ func doDeployment(deploymentConfig engine.Deployment) error {
 		return nil
 	}
 }
+
+func runDeploymentScript(context string, deploymentCommand []string, version string, p func(string, ...any)) error {
+	p("running %s-deployment command %s...", context, strings.Join(deploymentCommand, " "))
+	deploymentCommand = append([]string{"bash"}, deploymentCommand...)
+	cmd := exec.Command("/usr/bin/env", deploymentCommand[0:]...)
```

Comment:
isn't `deploymentCommand[0:]...` the same as `deploymentCommand...`?

Reply:
Fair point